tools/docs/manager.ts

```diff
@@ -173,30 +81,9 @@ sidebar_label: ${displayName}
     if (manager !== 'regex') {
       md += '\n## Additional Information\n\n';
     }
-    md += managerReadmeContent + '\n\n';
-
-    const [featureList] = stringifyIssues(managerIssuesMap[manager]?.features);
-    if (featureList) {
-      md += '## Open feature requests\n\n';
-      md += featureList;
-      md += '\n';
-    }
+    md += managerReadmeContent;
```

Comment:
```suggestion
    md += `${managerReadmeContent}\n\n`;
```
what about the removed newlines?

Reply:
I moved the new lines into `generateFeatureAndBugMarkdown` so that it's consistent across the three page types